tools/build/CreateRelease.php

```diff
@@ -27,6 +27,7 @@
 require_once 'Exception/BuildException.php';
 require_once 'Library/ReleaseCreator.php';
 require_once 'Library/ConsoleWriter.php';
+require_once 'Library/Version.php';
```

Comment:
Looks like you forgot this file.

Reply:
True, maybe because this folder is ignored? I had to add -f it